src/Paysera/Bundle/CodeGeneratorBundle/Service/TypeDefinitionBuilder/TypeDefinitionBuilderInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Paysera\Bundle\CodeGeneratorBundle\Service\TypeDefinitionBuilder;
+
+use Paysera\Bundle\CodeGeneratorBundle\Entity\Definition\TypeDefinition;
+
+interface TypeDefinitionBuilderInterface
+{
+    /**
+     * @param string $name
+     * @param array $definition
+     *
+     * @return bool
+     */
+    public function supports(string $name, array $definition);
```

Comment:
annotations aren't necessary if return type is declared in method itself `public function supports(string $name, array $definition) : bool;`